### PR TITLE
set proper version of data files and update location of default versions

### PR DESCRIPTION
**Pull request by WeirdMachine**

### CODE REVIEW

File: content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md [hunk 1]
```diff
@@ -14,5 +14,5 @@ The following is an example configuration, showing all possible options. Note th
 don't define explicitly are always defaulted to these values.
 
 ```yaml
-{{< readfile "kubermatic/master/data/kubermaticConfiguration.yaml" >}}
+{{< readfile "kubermatic/v2.16/data/kubermaticConfiguration.yaml" >}}
```
Comment (kron4eg) [inline content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md:17]: There is even better way to handle this in simple cases that is future proof.

Copy kubermatic/v2.16/data/kubermaticConfiguration.yaml right next to this file, and change diff to:

```suggestion
{{< readfile "kubermaticConfiguration.yaml" >}}
```

That will ensure future uses (made by recursive copying).

**Comment by WeirdMachine** [comment]
/hold 
Waiting for https://github.com/kubermatic/kubermatic/pull/7394

File: content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md [hunk 2]
```diff
@@ -14,5 +14,5 @@ The following is an example configuration, showing all possible options. Note th
 don't define explicitly are always defaulted to these values.
 
 ```yaml
-{{< readfile "kubermatic/master/data/kubermaticConfiguration.yaml" >}}
+{{< readfile "kubermaticConfiguration.yaml" >}}
```
Comment (kron4eg) [inline content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md:17]: I mean, now you need to move `data/kubermatic/master/data/kubermaticConfiguration.yaml` to `content/kubermatic/v2.16/concepts/kubermaticconfiguration/`

And do the same operation with all other files that we can see in this patch.

Comment (WeirdMachine) [reply content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md]: Are you sure? I mean the master is likely to be changed especially for v2.16 right?
I think i just update those files with the one in the KKP repository and their corresponding release branch, which should be the source of truth.

File: content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md (hunk 1 above)
Comment (WeirdMachine) [reply content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md:17]: According to the docs of Hugo the  filename must be relative to the current project working directory, or the project’s /content folder.

**Review by kron4eg — APPROVED** [review]

**Pull request merged**